ceph.spec.in

```diff
@@ -61,6 +78,11 @@ Requires: systemd
 %if 0%{with cephfs_java}
 BuildRequires: sharutils
 %endif
+%if 0%{with selinux}
+BuildRequires:	checkpolicy
+BuildRequires:	selinux-policy-devel
+BuildRequires:	/usr/share/selinux/devel/policyhelp
```

Comment:
@branto1 i still this error. `selinux-policy-mls` does not help.

see http://paste2.org/nBc4bxY7